src/edge/edgeManager.ts

```diff
@@ -452,6 +452,13 @@ export class EdgeManager {
                     "-B");
                 break;
             default:
+                const thirdPartyModuleTemplate = this.get3rdPartyModuleTemplateByName(template);
+                if (thirdPartyModuleTemplate) {
+                    const command = thirdPartyModuleTemplate.command
+                        .replace(new RegExp(`\\${Constants.moduleNameSubstitution}`, "g"), name)
+                        .replace(new RegExp(`\\${Constants.repositoryNameSubstitution}`, "g"), repositoryName);
+                    await Executor.executeCMD(outputChannel, command, { cwd: `${parent}`, shell: true }, "");

```

Comment:
AFAIK, on Linux and macOS, Maven and Yeoman have issues with `shell: true` option.